smart-send-logistics/block.json

```
@@ -0,0 +1,13 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
+	"name": "create-block/shipping-workshop",
```

This name does not follow the specs: https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#name

> The name for a block is a unique string that identifies a block. Names have to be structured as `namespace/block-name`, where namespace is the name of your plugin or theme.

```suggestion
	"name": "smart-send/pick-up-points",
```